tests/neg/i3012/a.scala

```diff
@@ -0,0 +1,5 @@
+
+object a extends fuz.Fuzbar {
+  override def str = ""
+  str()()()()()()  // error: missing argument
```

Comment:
Add a line with just `str()()` to check that it's also an error? A pos test with `str` and `str()` would be useful too.